docs/getting-started-guides/rkt/index.md

```diff
@@ -1,29 +1,25 @@
 ---
 ---
 
-This document describes how to run Kubernetes using [rkt](https://github.com/coreos/rkt) as a container runtime.
+This document describes how to run Kubernetes using [rkt](https://github.com/coreos/rkt) as the container runtime.
 
-### Prerequisite
+### Prerequisites
 
-- [systemd](http://www.freedesktop.org/wiki/Software/systemd/) should be installed on the machine and should be enabled.
-  The minimum version required for Kubernetes 1.3 is `219`.
-  *(Note that systemd is not required by rkt itself, we are using it here to monitor and manage the pods launched by kubelet.)*
+* [Systemd](http://www.freedesktop.org/wiki/Software/systemd/) must be installed and enabled. The minimum systemd version required for Kubernetes v1.3 is `219`. Systemd is used to monitor and manage the pods on each node.
 
-- Install the latest rkt release according to the instructions [here](https://github.com/coreos/rkt).
-  The minimum version required is [v1.9.1](https://github.com/coreos/rkt/releases/tag/v1.9.1).
+* [Install the latest rkt release](https://coreos.com/rkt/docs/latest/trying-out-rkt.html). The minimum rkt version required is [v1.9.1](https://github.com/coreos/rkt/releases/tag/v1.9.1). The [CoreOS Linux alpha channel](https://coreos.com/releases/) ships with a recent rkt release, and you can easily [upgrade rkt on CoreOS](https://coreos.com/rkt/docs/latest/install-rkt-in-coreos.html), if necessary.
 
-- The [rkt API service](http://coreos.com/rkt/docs/latest/subcommands/api-service.html) must be running on the node.
+* The [rkt API service](https://coreos.com/rkt/docs/latest/subcommands/api-service.html) must be running on the node.
 
-### Setup network
+### Set up pod networking
 
-You can configure the Kubernetes networking using its own `kubenet` and `CNI` [network
-plugins](http://kubernetes.io/docs/admin/network-plugins/) by setting the kubelet's `--network-plugin` and `--network-plugin-dir` flag.
-In addition, rkt supports using rkt's [Contained Networking](https://coreos.com/rkt/docs/latest/networking.html#contained-mode).
+You can configure Kubernetes pod networking using the usual Kubernetes `kubenet` and `CNI` [network plugins](http://kubernetes.io/docs/admin/network-plugins/) by setting the kubelet's `--network-plugin` and `--network-plugin-dir` options appropriately.
 
-##### Use rkt's Contained Networking
+In addition, rkt supports using its own [*contained network*](https://coreos.com/rkt/docs/latest/networking/overview.html#contained-mode), flannel SDN networking, or some provider networks.
```

Comment:
what about: 

In addition, rkt supports using its own [*contained network*](https://coreos.com/rkt/docs/latest/networking/overview.html#contained-mode), potentially layered above the flannel SDN, or some provider networks.